src/styles/MovieBanner.scss

```diff
@@ -73,16 +98,18 @@
     @media (min-width: 768px) {
       display: flex;
       flex-direction: column;
+      justify-content: center;
       align-items: flex-start;
       margin-left: 26px;
     }
 
-    @media screen and(min-width: 992px) {
```

Comment:
Please, use '@mixin for-desktop' from variables as well

Reply:
Thanks for a hint.